### DISCUSSION TITLE
How to book interest received on a private banking account?

### DISCUSSION THREAD

**Discussion by vctrade** [ghostfolio/ghostfolio#3075] [Q&A]
Hi All,

How to book interest received on a private banking account?

**Comment by dtslvr** [comment]
Hi @vctrade

You can create an activity of type _Interest_ for that and link it with your private banking account (see _Portfolio_ → _Activities_ → _Add activity_).

**Comment by vctrade** [comment]
> Hi @vctrade
> 
> 
> 
> You can create an activity of type _Interest_ for that and link it with your private banking account (see _Portfolio_ → _Activities_ → _Add activity_).

Hi Thomas,

Yes tried that but the checkbox (update cash balance) is greyed out. So using that function does not add that amount to the account itself. But the whole saving account is also not visible in the overview and analysis section.

**Reply by dtslvr** [reply]
That functionality is currently missing. Let me know if you are interested in implementing it.

You can check the total amount of interest via _Overview_ → _Summary_.

**Reply by vctrade** [reply]
Hi Thomas,

2 essential features that I'm missing which would be highly appreciated if implemented. 

1) booking interest, add that to the cash of the correspond account and being able to have all accounts part of the performance. E.g DEGIRO, raisin, ibkr but also the savings account of a bank. In the end all those combined make up you total performance. 
2) be able to create your own instrument if this is unavailable. Such as government bonds with a discounted purchase value and a face value. 

Let me know if you need more information. 

Br
Vincent

**Reply by vctrade** [reply]
Hi Thomas, can you add these features to the wishlist? Or do you want me to create an issue?

**Reply by dtslvr** [reply]
> the checkbox (update cash balance) is greyed out

I have created an issue: https://github.com/ghostfolio/ghostfolio/issues/3142

> be able to create your own instrument

This is already possible:

1. Open the _Admin Control_ panel
2. Go to _Market Data_
3. Create an asset profile (_Add manually..._)

**Reply by davidojedalopez** [reply]
Hey there 👋 

I couldn't find this option. I'm using the premium version. Do you mind sharing a picture of those settings?
> This is already possible:
> 1. Open the Admin Control panel
> 2. Go to Market Data
> 3. Create an asset profile (Add manually...)

**Reply by dtslvr** [reply]
> I couldn't find this option.

You cannot access the _Admin Control_ panel in the Ghostfolio cloud. An alternative is creating _Valuable_ activities.
